dockerfile-examples/Dockerfile.node-22.9.0-alpine-3.20-with-native

```diff
@@ -1,7 +1,7 @@
 # An Alpine image with additional Alpine packages, so native addons can
 # be compiled via node-gyp.
 
-FROM node:16.13.0-alpine3.14
+FROM node:22.9-alpine3.20
```

Comment:
Same here.

I assume the difference between these two Dockerfiles is the python3 install?